src/function/mtk.jl

```diff
@@ -35,11 +54,23 @@ function instantiate_function(f, x, adtype::AutoModelingToolkit, p, num_cons=0)
         hv = f.hv
     end
 
+    expr = symbolify(ModelingToolkit.Symbolics.toexpr(ModelingToolkit.equations(sys)))
```

Comment:
Does `symbolify` convert `x(t)` to `x`?

Reply:
Variables in an OptimizationSystem shouldn't have dependencies.

Reply:
>Does symbolify convert x(t) to x?

No, it only converts operators to symbols